### PR TITLE
Add UUIDs in ws_object

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Added UUID support in ws_object including callbacks (and
NULL-initialization in non-ws_object types).

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@neithernut `libuuid` for the CI missing here.

**Comment by neithernut** [comment]
Why do you nofity me? Notify @TheNeikos.

**Comment by matthiasbeyer** [comment]
-.- It's toooooo laaaaaateeee....

**Comment by neithernut** [comment]
I think this can wait until tomorow... or the day after.

**Comment by TheNeikos** [comment]
Okay, so it is installed on the image.

However, there is a problem with the location of the headers. @neithernut we will have to look at it together.

**Comment by matthiasbeyer** [comment]
It doesn't build on my machine, though.

**Comment by neithernut** [comment]
I'll have a look at it as soon as I'm with you guys.

File: cmake/Modules/FindUUID.cmake [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#  UUID_LIBRARIES - The libraries needed to use libuuid
+#  UUID_DEFINITIONS - Compiler switches required for using libuuid
+
+find_package(PkgConfig)
+pkg_check_modules(PC_UUID QUIET uuid)
+
+set(UUID_DEFINITIONS ${PC_UUID_CFLAGS_OTHER})
+
+find_path(UUID_INCLUDE_DIR uuid/uuid.h
+    HINTS ${PC_UUID_INCLUDEDIR} ${PC_UUID_INCLUDE_DIRS})
```
Comment (neithernut) [inline cmake/Modules/FindUUID.cmake]: Try to add `PATH_SUFFIXES ..`

**Comment by neithernut** [comment]
ACK.

**Pull request merged**